docs/gaia-tutorials/join-mainnet.md

```diff
@@ -132,13 +132,17 @@ The initial recommended `min-gas-prices` is `0.025uatom`, but you might want to
 
 ## Pruning of State
 
-There are three strategies for pruning state, please be aware that this is only for state and not for block storage:
+There are four strategies for pruning state, please be aware that this is only for state and not for block storage.
+Pruning can be set by either adjusting the `pruning` parameter in the `~/.gaia/config/app.toml` file.
+The following settings are available
 
-1. `PruneEverything`: This means that all saved states will be pruned other than the current.
-2. `PruneNothing`: This means that all state will be saved and nothing will be deleted.
-3. `PruneSyncable`: This means that only the state of the last 100 and every 10,000th blocks will be saved.
+1. `everything`: This means that all saved states will be pruned other than the current.
+2. `nothing`: This means that all state will be saved and nothing will be deleted.
+3. `default`: This means that only the state of the last 100 and every 10,000th blocks will be saved.
+4. `custom`: Pruning settings can be specified through `pruning-keep-recent`, `pruning-keep-every`, and `pruning-interval`
 
-By default every node is in `PruneSyncable` mode. If you would like to change your nodes pruning strategy then you must do so when the node is initialized. For example, if you would like to change your node to the `PruneEverything` mode then you can pass the `---pruning everything` flag when you call `gaiad start`.
+By default every node is in `default` mode. This is the recommended setting for most environments.
+If you would like to change your nodes pruning strategy then you must do so when the node is initialized. Passing a flag when starting `gaia` will always override settings in the `app.toml` file, if you would like to change your node to the `everything` mode then you can pass the `---pruning everything` flag when you call `gaiad start`.
```

Comment:
```suggestion
You can only change your node pruning strategy when the node is initialized. Passing a flag when starting `gaia` always overrides the settings in the `app.toml` file. For example, to change your node to the `everything` mode, you can pass the `---pruning everything` flag when you issue the `gaiad start` command:

`gaiad start ---pruning everything`
```

Reply:
let's show the command to change the pruning strategy, is this ^^ correct?

Reply:
In order to change pruning settings, you must reset your current database with `gaiad unsafe-reset-all` or `gaiad init`